backend/closure_service.py

```
@@ -0,0 +1,173 @@
+    
+    # Configuration
+    CONFIRMATION_THRESHOLD = 0.60  # 60% of followers must confirm
+    TIMEOUT_DAYS = 7  # 7 days to confirm
+    MINIMUM_FOLLOWERS = 3  # Minimum followers needed for confirmation process
+    
+    @staticmethod
+    def request_closure(grievance_id: int, db: Session) -> dict:
+        """Request closure for a grievance - triggers confirmation process"""
+        grievance = db.query(Grievance).filter(Grievance.id == grievance_id).first()
+        if not grievance:
+            raise ValueError("Grievance not found")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)